### PR TITLE
Network policy

### PR DESCRIPTION
**Pull request by rgstephens**
Generate NetworkPolicies for all ingress/egress service combinations. Issue #160. Issue [comment](https://github.com/RasaHQ/rasa-x-helm/issues/160#issuecomment-812229033) describes use.

### CODE REVIEW

**Review by virtualroot — APPROVED** [review]

File: charts/rasa-x/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 apiVersion: v2
 
-version: "1.8.2"
+version: "1.8.4"
```
Comment (virtualroot) [inline charts/rasa-x/Chart.yaml:4]:
```suggestion
version: "1.8.6"
```

**Review by tczekajlo — APPROVED** [review]
One small comment, the rest of it LGTM

**Comment by rgstephens** [comment]
Added db migration to Postgress ingress/egress policies

**Comment by tczekajlo** [comment]
@rgstephens I've found several things to fix. In the last commit, I pushed all fixes 😄

**Pull request merged**